dist/ami/files/scylla_install_ami

```diff
@@ -63,15 +67,14 @@ if __name__ == '__main__':
 
     run('yum update -y')
 
-    run('curl -L -o /etc/yum.repos.d/scylla-ami-drivers.repo https://copr.fedorainfracloud.org/coprs/scylladb/scylla-ami-drivers/repo/epel-7/scylladb-scylla-ami-drivers-epel-7.repo')
     if args.repo_for_install:
         run('curl -L -o /etc/yum.repos.d/scylla_install.repo {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
 
     if args.localrpm:
-        rpms = glob.glob('/home/centos/scylla*.*.rpm')
+        rpms = glob.glob('{}/scylla*.*.rpm'.format(homedir))
         run('yum install -y {}'.format(' '.join(rpms)))
     else:
-        run('yum install -y scylla-ami scylla-debuginfo')
```

Comment:
why removing installation of scylla-ami ?

Reply:
@syuu1228 ^

Reply:
Ah, this was caused by mistake. I wasn't hit error because I tested with --localrpm option.
I'll remove this change.

Reply:
It reverted at https://github.com/scylladb/scylla/commit/efd0fc9f769e9e6020031bbd9b6bc399e5219800